src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -21,7 +21,11 @@ public partial class HttpsTests : ConditionalWcfTest
                      nameof(SSL_Available))]
 #endif
     [OuterLoop]
-    public static void Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerTrust_EchoString()
+    [ActiveIssue(1381)] // requires additional SSL configuration of new port to work
```

Comment:
Reviewers: I'm adding this test but also disabling it with an ActiveIssue pending further discussion.  As far as I can tell, to get a different cert working end to end with https could require using a different port and therefore changes to our setup scripts, etc.  If we see sufficient value in this test, we can pursue it, otherwise I'll remove it.  The TCP equivalent of these tests has an end-to-end peer cert positive test that covers that part of the product.